packages/bpk-component-banner-alert/src/BpkBannerAlert-test.js

```diff
@@ -18,163 +18,41 @@
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with a element based message', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.SUCCESS}
-          message={
-            <p>
-              All <span>good</span>
-            </p>
-          }
-          toggleButtonLabel="View more"
-        >
-          {message}
-        </BpkBannerAlert>,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with a custom class name', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          className="custom-class"
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with a custom banner-alert class name', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          bannerClassName="custom-class"
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with arbitrary props', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          id="custom-id"
-          hidden="hidden"
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with dismissable option', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          dismissable
-          dismissButtonLabel="Dismiss"
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with show set false', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          dismissable
-          dismissButtonLabel="Dismiss"
-          show={false}
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with animateOnEnter', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          dismissable
-          dismissButtonLabel="Dismiss"
-          animateOnEnter
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('should render correctly with animateOnLeave', () => {
-    const tree = renderer
-      .create(
-        <BpkBannerAlert
-          type={ALERT_TYPES.WARN}
-          message={message}
-          dismissable
-          dismissButtonLabel="Dismiss"
-          animateOnLeave
-        />,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
+  it('should render correctly', () => {
```

Comment:
I opted for shallow rendering since `BpkBannerAlertInner` is already well tested and these components just delegate to it